libp2p/peer/peermetadata_interface.py

```diff
@@ -1,13 +1,20 @@
 from abc import ABC, abstractmethod
+from typing import (
+    Any,
+)
+
+from .id import (
+    ID,
+)
 
 
 class IPeerMetadata(ABC):
 
-    def __init__(self):
+    def __init__(self) -> None:
         pass
 
     @abstractmethod
-    def get(self, peer_id, key):
+    def get(self, peer_id: ID, key: str) -> Any:
```

Comment:
Note: Actually I'm not so sure if it should be `str` or `bytes`. Let me confirm that.